dlc/src/main/scala/org/bitcoins/dlc/BinaryOutcomeDLCClient.scala

```diff
@@ -272,9 +273,15 @@ case class BinaryOutcomeDLCClient(
                                        outputs = outputs,
                                        lockTime = UInt32.zero)
 
-    subtractFeeFromOutputs(txWithoutFee,
-                           feeRate,
-                           Vector(changeSPK, remoteChangeSPK))
+    val txWithFee = subtractFeeFromOutputs(txWithoutFee,
+                                           feeRate,
+                                           Vector(changeSPK, remoteChangeSPK))
+
+    BaseTransaction(version = txWithFee.version,
+                    inputs = txWithFee.inputs,
+                    outputs =
+                      txWithFee.outputs.filter(_.value >= Policy.dustThreshold),
```

Comment:
doesn't this remove all outputs under dust, not just the change?

Reply:
yup :sunglasses: